plugins/bucketize_procedure.cc

```diff
@@ -224,7 +228,9 @@ run(const ProcedureRunConfig & run,
         DEBUG_MSG(logger) << "Bucket " << mappedRange.first << " from "
                           << lowerBound << " to " << higherBound;
 
-        parallelMap(lowerBound, higherBound, applyFct);
+        if (!parallelMapHaltable(lowerBound, higherBound, applyFct)) {
+            DEBUG_MSG(logger) << BucketizeProcedureConfig::name << " procedure was cancelled";
```

Comment:
You mean the logger message or some other notification?

I am not sure if there are a single place where I can log the message.  I will take a look.